app/controllers/application_controller.rb

```diff
@@ -1,5 +1,6 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  skip_before_action :verify_authenticity_token
```

Comment:
Wait, why?

Reply:
It's a form submission from the Touchstone system essentially so it does not have an authenticity token. It's from the docs.